.github/workflows/dcpy_test.yml

```diff
@@ -15,8 +15,23 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: (Temp) Install stubs # can remove once these make it into base docker image
-        run: pip install -r python/requirements.txt
-
       - name: Static Type Check (mypy)
         run: mypy ./dcpy
+
+  unit_tests:
+    name: Unit tests
+    runs-on: ubuntu-22.04
+    container:
+      image: nycplanning/build-geosupport:latest
+    env:
+      BUILD_ENGINE_DB: PLACEHOLDER_VALUE
+      BUILD_ENGINE_SERVER: PLACEHOLDER_VALUE
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Run python tests
+        run: |
+          python3 -m pytest ./dcpy/test/ -s --verbose --verbose --cov-config=pyproject.toml --cov .
```

Comment:
this gives us [a test coverage report](https://github.com/NYCPlanning/data-engineering/actions/runs/6250559203/job/16969711745#step:4:31) which can be configured to be more accurate ([example](https://github.com/NYCPlanning/data-engineering/actions/runs/6250559203/job/16969711076#step:4:66)) using the pyproject.toml ([example config](https://github.com/NYCPlanning/data-engineering/blob/main/products/template/pyproject.toml#L9))

I didn't add such configurations in this PR